### ISSUE TITLE
Unbalanced transition warning On iOS9 

### ISSUE THREAD

**Issue by AnthonyMDev** [BranchMetrics/iOS-Deferred-Deep-Linking-SDK#182]
On the first launch of the application, while using Branch I get a warning in the debug log

````
Unbalanced calls to begin/end appearance transitions for <UIViewController: 0x7f8fe06f2a70>.
````

I'm pretty sure this is due to the `SFSafariViewController` that you are presenting in your own `UIWindow`.

**Comment by aaustin** [comment]
Hey @AnthonyMDev  - we've got a patch in this PR this waiting on the next release: https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK/pull/186

We'll wait to accumulate a few other fixes before doing another release though. Feel free to tweak the code if it's pestering you.
